registration/views.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth import authenticate, login, logout
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect
+from django.shortcuts import render
+from django.urls import reverse
+
+# Create your views here.
+def login_view(request: HttpRequest) -> HttpResponse:
+    if request.method == 'POST':
+        username = request.POST['username']
+        password = request.POST['password']
+        user = authenticate(request, username=username, password=password)
+        if user is not None:
+            login(request, user)
+            return HttpResponseRedirect(reverse('taxi:index'))
```

Comment:
Ensure that the URL name 'taxi:index' is correctly defined in your project's URL configuration. If 'taxi' is not the correct namespace or app name, this could lead to a `NoReverseMatch` error.